src/DCGView.ts

```diff
@@ -101,6 +101,15 @@ declare global {
       strong: any;
       ul: any;
       li: any;
+      table: any;
+      tr: any;
+      th: any;
+      td: any;
+      button: any;
+      svg: any;
+      title: any;
+      path: any;
```

Comment:
Thought you said `svg` and `path` don't work?

Reply:
You're right, they don't--- I forgot to remove these.